src/both/DeltaNeutralLpV3.sol

```diff
@@ -131,6 +144,36 @@ contract DeltaNeutralLpV3 is AccessStrategy {
     /// @dev Aave deposit receipt token.
     ERC20 public immutable aToken;
 
+    /*//////////////////////////////////////////////////////////////
+                          PRICE CONVERSION
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice abs(asset.decimals() - borrow.decimals() - borrowFeed.decimals()). This value has significance
+     * while converting asset amount to borrow amount and vice versa.
+     */
+    uint256 public decimalAdjust;
+    /**
+     * @notice true if `asset.decimals() - borrow.decimals() - borrowFeed.decimals()` is positive. false
+     * otherwise.
+     */
```

Comment:
This is actually true if the expression is >= 0. You can make this into a single line comment as well.